pages/glossary/semantic-role.md

```diff
@@ -7,15 +7,15 @@ input_aspects:
   - DOM tree
 ---
 
-A _semantic [role](https://www.w3.org/TR/wai-aria-1.1/#dfn-role)_ is a semantic association that indicates an object's type. This allows tools to present and support interaction with the object in a manner that is consistent with user expectations about other objects of that type.
+A _semantic association_, also called [role][], indicates an object's type. This allows tools to present and support interaction with the object in a manner that is consistent with user expectations about other objects of that type.
```

Comment:
Adding semantic association made this more complicated then I think this needs to be. Why not just define explicit role based on the role attribute, implicit role based on AAM, and semantic role on explicit or implicit role?

Reply:
Yes, maybe the good way to do it is to kill this notion rather than try to make it obvious.